src/index.js

```diff
@@ -0,0 +1,54 @@
+import './index.css';
+
+const taskInput = document.querySelector('input');
+const taskContainer = document.querySelector('.task-container');
+
+class Task {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+const taskArray = [];
```

Comment:
- [ ] As requested by the previous reviewer 👇🏻 

> Although you are required to display a static list, you've gone a step further to implement adding user entered items. However, the list index is required to begin from 1. Consider removing the - 1 in the snippet addToLocalStorage(task, false, checkBox.length - 1);

- [ ] Make sure to address these requested changes 😄 

- You should have an array of some simple *to do tasks* (array of objects) with each object containing three keys - description \[`string`\], completed \[`bool`\], index: \[`number`\]. 
- Then write a function that iterates over the tasks array and populates the HTML list item element for each task.
- The task array is rendered dynamically to the DOM when the page load and don't forget 🤔 as requested by the previous reviewer the tasks should appear in order of the `index` values for each task

Reply:
The requested changes were already implemented.
![todolist](https://user-images.githubusercontent.com/94173744/172856762-26baa81c-88f9-493e-8127-b17c1a885dad.PNG)